docs/data-modeling/metrics.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Metrics
+redirect_from:
+  - /docs/latest/administration-guide/07-segments-and-metrics
+  - /docs/latest/data-modeling/segments-and-metrics
+---
+
+# Metrics
+
+Create metrics to define the official way to calculate important numbers for your team.
+
+Metrics are like pre-defined calculations: create your aggregations once, save them as metrics, and use them whenever you need to analyze your data. They're like [models](./models.md), but for single values like a count, average, or sum.
+
+For example, you may want to create a metric that calculates revenue, so people can refer to revenue in their own questions. That way you standardize how revenue is calculated (so you don't end up with five different calculations for the revenue).
+
+## Using metrics
+
+You can:
+
+- Save metrics to [collections](../exploration-and-organization/collections.md).
+- Add metrics to [dashboards](../dashboards/introduction.md).
+- View metrics in the [data browser](../exploration-and-organization/exploration.md#browse-your-data).
+- Refer to metrics in questions.
+
+### Metrics in the query builder
+
+When asking questions in the query builder, you can find metrics that your team created in the summarization section under **Common metrics**.
+
+![Common metrics](./images/common-metrics.png)
+
```

Comment:
```suggestion

You can also pick a metric as a data source when creating a new question. In the query builder it will be shown in the same way. If time dimension for this metric is defined it will be added as a grouping. You can change the groupings to break out the metric by other dimensions.

**TODO** Picture of the entity picker with Metrics tab opened

You can use custom expressions editor to combine multiple metrics or perform any calculations on top of them.

**TODO** Picture like metrics-built-on-other-metrics.png but from the question editor

```